modules/sqs-queue/modules/terraform-aws-sqs-queue/main.tf

```diff
@@ -0,0 +1,24 @@
+locals {
```

Comment:
why do we need a module inside the module if the code is simple and consists of just one resource?

Reply:
that's a good point, can we defer it to another PR though? as that will increase the scope of this PR

Reply:
@aknysh this is because we tried to upstream the module and it never got off the ground. Then someone deleted the upstream repo while it was still incubating...

Reply:
There are some old slack threads on it. IIRC, it was because of the `string` type vs `list(string)` type arguments around the `count` / `for_each` logic.